src/delfino/main.py

```diff
@@ -83,17 +90,76 @@ def invoke(self, ctx: click.Context) -> Any:
     def get_help(self, *args, **kwargs) -> str:
         help_str = super().get_help(*args, **kwargs)
 
-        if self._hidden_plugins:
+        if self._hidden_plugin_commands:
             if logging.root.level == logging.DEBUG:
                 help_str += click.style(
-                    f"\n\nDisabled command{'s' if len(self._hidden_plugins) > 1 else ''}: "
-                    + ", ".join(sorted(self._hidden_plugins))
-                    + f" (see 'tool.{ENTRY_POINT}.disable_commands' in '{PYPROJECT_TOML_FILENAME}')",
+                    f"\n\nDisabled command{'s' if len(self._hidden_plugin_commands) > 1 else ''}: "
+                    + ", ".join(sorted([f"{plugin}.{cmd}" for plugin, cmd in self._hidden_plugin_commands.items()]))
+                    + f" (see 'tool.{ENTRY_POINT}.disable_plugin_commands' in '{PYPROJECT_TOML_FILENAME}')",
                     fg="white",
                 )
 
         return help_str
 
+    def _get_commands(self) -> Dict[str, click.Command]:
+        if self._commands:
+            return self._commands
+
+        command_packages = [
+            CommandPackage(package=self.PRESET_COMMAND_PACKAGE, plugin_name="delfino"),
+            *list(discover_packages()),
```

Comment:
This is the only place where `discover_packages` is used (not considering tests). So the use can be simplified:
- No need to yield. It can return a list because that's what you want here.
- The `plugin_group_name` argument can be removed because it is never set to a non-default value.
```suggestion
            *discover_packages(),
```

Reply:
This is also a legacy function because ultimately everything will be a plugin. The `src/delfino/commands` folder won't exist.